implementations/rust/ockam/ockam_api/src/nodes/service/node_services.rs

```diff
@@ -184,6 +185,12 @@ impl NodeManager {
             return Err(ApiError::generic("Hop service exists at this address"));
         }
 
+        ctx.flow_controls().add_consumer(
+            addr.clone(),
+            &self.api_transport.flow_control_id,
+            FlowControlPolicy::SpawnerAllowMultipleMessages,
+        );
```

Comment:
What happens if this is called twice? Should it fail or ignore that it already exists?

Reply:
It's idempotent